dm/tests/shardddl3_1/run.sh

```diff
@@ -532,16 +532,23 @@ function DM_117_CASE {
 
 	run_dm_ctl_with_retry $WORK_DIR "127.0.0.1:$MASTER_PORT" \
 		"query-status test" \
-		'ALTER TABLE `shardddl`.`tb` ADD COLUMN `b` INT' 1 \
+		"because schema conflict detected" 1 \
 		"add column b that wasn't fully dropped in downstream" 1
 	run_dm_ctl_with_retry $WORK_DIR "127.0.0.1:$MASTER_PORT" \
 		"shard-ddl-lock" \
 		'ALTER TABLE `shardddl`.`tb` ADD COLUMN `b` INT' 1
 
 	# try to fix data
+	echo 'create table tb1(a int primary key, b int, c int) engine=innodb default charset=latin1 collate=latin1_bin;' >${WORK_DIR}/schema.sql
+	run_dm_ctl $WORK_DIR "127.0.0.1:$MASTER_PORT" \
+		"binlog-schema update test ${shardddl1} ${tb1} ${WORK_DIR}/schema.sql -s mysql-replica-01" \
+		"\"result\": true" 2
+
+	# skip this error
 	run_dm_ctl_with_retry $WORK_DIR "127.0.0.1:$MASTER_PORT" \
-		'shard-ddl-lock unlock "test-`shardddl`.`tb`" -s mysql-replica-01 --action exec -d shardddl1 -t tb1' \
-		"\"result\": true" 1
+		"binlog skip test" \
+		"\"result\": true" 2 \
+		"\"source 'mysql-replica-02' has no error\"" 1
```

Comment:
How about check the task status after skip? Maybe insert some new rows.

Reply:
https://github.com/pingcap/tiflow/pull/4883/commits/02270f79808d88bd6b2d0af0e1fa3843cc9af326